### PR TITLE
[BBC-9121] MDX MediaContentSection

### PR DESCRIPTION
**Pull request by mcellier**
## Description

Migrate `MediaContentSection` to the new mdx format

## How it was tested

Locally with storybook + homepage on kairos

### CODE REVIEW

File: src/layout/section/mediaContentSection/MediaContentSection.story.mdx [hunk 1]
```diff
@@ -31,5 +30,17 @@ stories.add('default', () => (
...
-))
+  </Story>
+</Preview>
+
+## Specifications
+
+## Usage
+
+```jsx
+import { MediaContentSection } from '@blablacar/ui-library/build/MediaContentSection'
```
Comment (gabrielagabriel) [inline src/layout/section/mediaContentSection/MediaContentSection.story.mdx:41]:
```suggestion
import { MediaContentSection } from '@blablacar/ui-library/build/mediaContentSection'
```

**Review by gabrielagabriel — APPROVED** [review]
🎉

File: src/layout/section/mediaContentSection/index.tsx [hunk 2]
```diff
@@ -1,60 +1,2 @@
...
-      align-items: center;
-    }
-  }
-`
-
-export { MediaContentSectionProps } from './mediaContentSection'
-export { StyledMediaContentSection as MediaContentSection }
-export default StyledMediaContentSection
+export { MediaContentSection, MediaContentSectionProps } from './MediaContentSection'
+export { MediaContentSection as default } from './MediaContentSection'
```
Comment (gabrielagabriel) [inline src/layout/section/mediaContentSection/index.tsx:2]:
```suggestion
export { MediaContentSection as default } from './mediaContentSection'
```

File: src/layout/section/mediaContentSection/index.tsx [hunk 3]
```diff
@@ -1,60 +1,2 @@
...
-    & .kirk-media-content-wrapper {
-      align-items: center;
-    }
-  }
-`
-
-export { MediaContentSectionProps } from './mediaContentSection'
-export { StyledMediaContentSection as MediaContentSection }
-export default StyledMediaContentSection
+export { MediaContentSection, MediaContentSectionProps } from './MediaContentSection'
```
Comment (gabrielagabriel) [inline src/layout/section/mediaContentSection/index.tsx:1]:
```suggestion
export { MediaContentSection, MediaContentSectionProps } from './mediaContentSection'
```

File: src/layout/section/mediaContentSection/MediaContentSection.story.mdx (hunk 1 above)
Comment (mcellier) [reply src/layout/section/mediaContentSection/MediaContentSection.story.mdx]: Actually it's `import { MediaContentSection } from '@blablacar/ui-library/build/layout/section/mediaContentSection'`

**Pull request merged**